lib/segment/src/index/hnsw_index/graph_layers_builder.rs

```diff
@@ -655,18 +693,24 @@ mod tests {
         }
         let graph_layers = graph_layers_builder.into_graph_layers();
 
-        let number_layers = graph_layers.links_layers.len();
+        let number_layers = graph_layers.links.num_points();
```

Comment:
It is number of points or number of layers?

Reply:
In dev `graph_layers.links_layers.len()` is definitely a number of points and there is a mistake in the text message. The number of layers is present in the next `eprintln`.

Fixed message text and variable name.